binarytree/layout.py

```diff
@@ -0,0 +1,113 @@
+""" Module containing layout related algorithms."""
+
+
+def _get_coords(values):
```

Comment:
Since you are importing this outside of the module, should be renamed to `get_coords`.

Reply:
This is only used on layout module and test.